identity/handler_test.go

```diff
@@ -1492,51 +1488,42 @@ func TestHandler(t *testing.T) {
 				t.Parallel()
 				body, res := getFull(t, ts, fmt.Sprintf("/identities?per_page=%d", perPage), http.StatusOK)
 				assert.Len(t, body.Array(), perPage)
-				assert.Equal(t, strconv.Itoa(count), res.Header.Get("X-Total-Count"))
+				_ = res
```

Comment:
Reason for removal is we no longer send `X-Total-Count`?

```suggestion
```

Reply:
`SELECT COUNT(*)` expensive and deprecated. We don't want to support it in the new pagination API anymore.

Reply:
See https://github.com/cockroachdb/cockroach/issues/79579 for more context on why we cannot support this anymore.

Reply:
No, I get it, just wanted to make sure this was intended. 

But the `_ = res` can probably be removed?

Thanks for the explanation, though.

Reply:
Fixed and added assertion that `X-Total-Count` does not appear with token pagination.